test/verify/run-tests

```diff
@@ -60,8 +60,7 @@ def finish_test(opts, test):
 
     Return (retry_reason, exit_code). retry_reason can be None or a string.
     """
-
-    if test.process.returncode != 1:
+    if test.process.returncode == 0:
```

Comment:
Note to self: This *maaaaaay* need a check for code 77 as well (skips), let's see if we rely on that.